File: tdd-katas/vending-machine/vending-machine-typescript/src/vending_machine_spec.ts
import { VendingMachine } from './vending_machine';
import { Choice } from './vending_machine';
import { Can } from './vending_machine';

describe("Vending machine", () => {

  it("delivers nothing when choice does not exist", () => {
    expect(new VendingMachine().deliver(Choice.BEER)).toEqual(Can.NOTHING);
  });

})

File: tdd-katas/vending-machine/vending-machine-java/src/test/java/VendingMachineSpec.java
import com.mscharhag.oleaster.runner.OleasterRunner;
import org.junit.runner.RunWith;
import static com.mscharhag.oleaster.runner.StaticRunnerSupport.*;
import static com.mscharhag.oleaster.matcher.Matchers.*;

@RunWith(OleasterRunner.class)
public class VendingMachineSpec {
  private VendingMachine vendingMachine;
  
  {
    describe("A vending machine", () -> {

      it("delivers nothing when the machine is choiceless", () -> {
        expect(new VendingMachine().deliver(Choice.BEER)).toEqual(Can.NOTHING);
      });
    });
  }
}